kubetest/main.go

```diff
@@ -47,6 +48,9 @@ var (
 	verbose      = false
 	timeout      = time.Duration(0)
 	boskos       = client.NewClient(os.Getenv("JOB_NAME"), "http://boskos")
+
+	// kubetestContext is the Context that should be used for timeout / cancellation
+	kubetestContext context.Context
```

Comment:
If we like the context approach, we should probably thread this through functions in preference to using a global.  But that is probably a separate PR.